ex1.py

```diff
@@ -0,0 +1,22 @@
+FUNCTION_STOP_CHAR = "q"
+
+def calculate_average():
+    counter = 0
+    sum = 0
+    num = input('enter num if you want to stop enter {} '.format(FUNCTION_STOP_CHAR))
+    while num != FUNCTION_STOP_CHAR:
+        try:
+            sum += float(num)
+            counter += 1
+            num = input("enter num ")
+        except:
```

Comment:
when excepting an exception specify the type of exception your catching:

except ValueError as exception: